lib/EvalDevToolModuleTemplatePlugin.js

```diff
@@ -15,7 +15,7 @@ class EvalDevToolModuleTemplatePlugin {
 
 	apply(moduleTemplate) {
 		moduleTemplate.plugin("module", (source, module) => {
-			const content = source.source();
+			const content = "{" + source.source() + "\n}";
```

Comment:
This is a visible change for everyone using the eval devtool. Not sure if it's worth worsen the experience for everyone because of a browser bug.